.gitignore

```diff
@@ -1 +1,2 @@
+*.swp
```

Comment:
Should we really have any .swp files layout around? This seems too platform dependent.